File: Server/src/commonStructures/networkDropAttributes.ts
import NetworkObjectDropAttributes from "./NetworkObjectDropAttributes";

export default class NetworkDropAttributes {
    public playerId:number;
    public objectsDropped:NetworkObjectDropAttributes[];
    constructor(playerId:number, objectsDropped:NetworkObjectDropAttributes[]) {
        this.playerId = playerId;
        this.objectsDropped = objectsDropped;
    }
}

File: Assets/Scripts/Network/CommonStructures/NetworkDropAttributes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Network
{
    /// <summary>
    /// Used to send the informations about the objects dropped by a player.
    /// </summary>
    public class NetworkDropAttributes
    {
        /// <summary>
        /// The id of the player.
        /// </summary>
        public int playerId;
        /// <summary>
        /// The list of objects dropped by the player.
        /// </summary>
        public NetworkObjectDropAttributes[] objectsDropped;
    }
}
